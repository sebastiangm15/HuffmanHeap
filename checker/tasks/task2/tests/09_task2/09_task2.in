18
8 OYZ2
14 UZM83
19 PAL70
22 RFK23
24 JXK35
39 NYL68
50 YMX75
55 NDF56
64 VFG76
65 MZH93
70 FZM72
70 XUE4
70 YQZ11
78 GHO57
89 EEL83
98 DIB4
98 NEE77
100 XON60
3
010101110001111111001001
1111010101010
010011100111011101101111110011111
