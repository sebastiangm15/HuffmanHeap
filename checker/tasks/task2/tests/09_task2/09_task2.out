XON60 YQZ11 MZH93 YMX75 GHO57 FZM72 
JXK35 XON60 XUE4 
XON60 VFG76 NEE77 OYZ2 UZM83 GHO57 YMX75 
